ADJ-Internship/DataModel/OrderTrack/Order.cs

```diff
@@ -40,38 +33,29 @@ public class Order : EntityBase
         [StringLength(30)]
         public string Department { get; set; }
 
-        //[Display(Name = "Order Type")]
         [StringLength(30)]
         public string OrderType { get; set; }
 
-        //Droplist from 2010 to 2020
         public string Season { get; set; }
 
         [StringLength(30)]
         public string Factory { get; set; }
 
-        //Default value = USD
-        public string Currency { get; set; } = "USD";
+        public enum Currency { USD, Euro, VND };
```

Comment:
create a file called Enums in Common project with 2 enums for Currency and Status